cargo-marker/README.md

```diff
@@ -0,0 +1,77 @@
+# Cargo Marker
+
+[![Crates.io](https://img.shields.io/crates/v/cargo-marker.svg)](https://crates.io/crates/cargo-marker)
+<!--
+FIXME(xFrednet): Add license shield, once crates.io also says:
+[![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/cargo_marker.svg)](#license)
+-->
+
+*cargo_marker* is the CLI tool for [Marker], an experimental linting interface for Rust. It seamlessly integrates with the Rust compiler to provide custom linting capabilities for your Rust projects.
+
+> **Note**
+>
+> The project is in the very early stages of development, some things are still missing and the API is not stable yet.
+
+[Marker]: https://github.com/rust-marker/marker
+
+## Key Features
+
+* **Seamless Integration**: *cargo_marker* effortlessly integrates with your existing workflow to leverage Marker's linting capabilities and ensure code quality.
+* **Automatic Lint-Crate Compilation**: *cargo_marker* automatically fetches and builds specified lint crates, streamlining the process of incorporating additional linting rules into your project.
+* **User-Friendly Setup**: *cargo_marker* can automatically set up the driver and toolchain, allowing you to focus on writing quality code. (This version will setup rustc's driver for `nightly-2023-06-01`)
+
+## Prerequisites
+
+*cargo_marker* requires *[Cargo]* and *[rustup]* to be installed. Currently, only Unix and Windows systems are supported. Linux, Windows, and macOS are actively tested in the CI.
+
+[Cargo]: https://github.com/rust-lang/cargo/
+[rustup]: https://github.com/rust-lang/rustup/
+
+## Usage
+
+### Installation
+
+```sh
+cargo install cargo_marker
+
+# Automatically setup the toolchain and driver
+cargo marker setup --auto-install-toolchain
+```
+
+### Specifying lints
+
+Marker requires lint crates to be specified. The best way, is to add them to the `Cargo.toml` file, like this:
+
+```sh
+[workspace.metadata.marker.lints]
+# Add a local crate as a path
+local_lint_crate = { path = "path/to/lint_crate" }
+# Add an external crate via git
+git_lint_crate = { git = "https://github.com/rust-marker/marker" }
+```
+
+Lints from registries, like crates.io, are sadly not yet supported. See [rust-marker/marker#87](https://github.com/rust-marker/marker/issues/87).
```

Comment:
Might be good to mention other limitations if they exist. Or mention potential cases that will be ignored or can cause error.

Reply:
I want to keep an "active" list of API limitations in the main readme. This one is listed here, as it's a limitation of `cargo-marker`. Once the crate is downloaded, `cargo-marker` could again handle everything. It might be worth adding a lint to the main readme, in the note box at the top.